ui/src/main/resources/data/GrantMatchAccess.xml

```diff
@@ -42,10 +42,9 @@
   #set ($accessRequest = $services.anonymousCommunication.grantAccess($request.id))
```

Comment:
Just curious if we ever use this file (`GrantMatchAccess.xml`), I couldn't find where it is used?

Reply:
not sure about this file as well

Reply:
It was used; in the email sent from the patient record's similar cases, the message was something like "hello, we have similar patients, here's my patient info, if you would like to collaborate, click here to grant me access to your patient". I see that is no longer happening, so this is probably unused.